dialogy/plugins/preprocess/text/calibration.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, List, Dict
+
+import math
+
+
+def predict_alternative(
+    alternative: Dict[str, Any], vectorizer: Any, classifier: Any
+) -> float:
+    vec_ = vectorizer.transform([alternative["transcript"]]).tolist()[0]
```

Comment:
let's split just once and use it?
[This](https://github.com/Vernacular-ai/dialogy/blob/2f688ad30ed14f220eb8ae1a7b82fbe04ad2edbf/dialogy/constants/__init__.py#L17) prevents typos and provides autocomplete.